ghcide/test/exe/Main.hs

```diff
@@ -6523,6 +6524,57 @@ expectSameLocations actual expected = do
                               return (filePathToUri fp, l, c))
     actual' @?= expected'
 
+hoverFixityTests :: TestTree
+hoverFixityTests = testGroup "Explicit fixity"
+    [ hoverTest "(++)" (Position 5 7) "infixr 5 `++`"
+    , hoverTest "($)" (Position 6 7) "infixr 0 `$`"
+    , hoverTest "(.)" (Position 7 7) "infixr 9 `.`"
+    , hoverTest "(+)" (Position 8 7) "infixl 6 `+`"
+    , hoverTest "(-)" (Position 9 8) "infixl 6 `-`"
+    , hoverTest "(<>)" (Position 10 7) "infixr 6 `<>`"
+    , hoverTest "(>>=)" (Position 11 7) "infixl 1 `>>=`"
+    , hoverTest "(>=>)" (Position 12 7) "infixr 1 `>=>`"
+    , hoverTest "elem" (Position 13 7) "infix 4 `elem`"
+    , hoverTest "on" (Position 14 7) "infixl 0 `on`"
+    , hoverTest "(||)" (Position 15 8) "infixr 2 `||`"
+    , hoverTest "mod" (Position 16 8) "infixl 7 `mod`"
+    , hoverTest "(**)" (Position 17 8) "infixr 8 `**`"
+    , hoverTest "(^)" (Position 18 8) "infixr 8 `^`"
+    , hoverTest "(<$)" (Position 19 8) "infixl 4 `<$`"
+    , hoverTest "seq" (Position 20 9) "infixr 0 `seq`"
+    , hoverTest "(<|>)" (Position 21 8) "infixl 3 `<|>`"
+    , hoverTest "fixity define" (Position 23 11) "infixr 7 `>>:`"
+    , hoverTest "record" (Position 28 10) "infix 9 `>>::`"
+    , hoverTest "wildcards" (Position 30 5) "infixr 7 `>>:`\n\ninfix 9 `>>::`"
+    , hoverTest "function" (Position 32 11) "infixl 1 `f`"
+    , hoverTest "signature" (Position 35 2) "infixr 9 `>>>:`"
+    , hoverTest "operator" (Position 36 2) "infixr 9 `>>>:`"
+    , hoverTest "escape" (Position 39 2) "infixl 3 `~\\:`"
+    -- It will cause error like: "Failed to load interface for \8216Fixity\8217\nIt is not a module in the current program, or in any known package."
```

Comment:
Get fixity from locally defined module failed because something like `Failed to load interface for \8216Fixity\8217\nIt is not a module in the current program, or in any known package.` It is strange.

It's a trivial case and I think we have an available function even without this.